src/main/resources/templates/search.html

```diff
@@ -10,14 +10,34 @@
 <title>Gas Mileage Calculator</title>
 </head>
 <body>
+<div th:replace="fragments/header :: header"></div>
 <div class="container">
-	<h2>Gas Mileage Calculator</h2>
-	<h4>Search for your car</h1>
+	<h4>Search for your car</h4>
 
     <form th:action="@{/search}" th:object="${carSearch}" method="post" class="form">
 	    <div class="form-group">
 	        <label for="carMake">Car Make</label>
-	        <input class="form-control" type="text" th:field="*{carMake}" id="carMake" placeholder="Make">
+	        <select class="form-control" th:field="*{carMake}" id="carMake">
+	        	<option value="Make">Make</option>
+	        	<option value="BMW">BMW</option>
+				<option value="Daewoo">Daewoo</option>
+	        	<option value="Ford">Ford</option>
+	        	<option value="Holden">Holden</option>
+	        	<option value="Honda">Honda</option>
+	        	<option value="Hyundai">Hyundai</option>
+	        	<option value="Isuzu">Isuzu</option>
+	        	<option value="Kia">Kia</option>
+	        	<option value="Lexus">Lexus</option>
+	        	<option value="Mazda">Mazda</option>
+	        	<option value="Mitsubishi">Mitsubishi</option>
+	        	<option value="Nissan">Nissan</option>
+	        	<option value="Peugeot">Peugeot</option>
+	        	<option value="Subaru">Subaru</option>
+	        	<option value="Suzuki">Suzuki</option>
+	        	<option value="Toyota">Toyota</option>
+	        	<option value="Volkswagen">Volkswagen</option>
+	        	<option value="Other">Other</option>
+	        </select>
```

Comment:
This change is completely up to you, but I think there may be some benefit in making this input a dropdown list. Making this a dropdown list will potentially reduce the input time for the form entry, and it will potentially help any validation on your end as only certain inputs would be allowed.

Reply:
I agree, Someone could just misspell a word and get the wrong result.
Plus this maybe over the top but, If you do make it a drop-down, when someone select Honda, make it so that it will only show the honda model